crates/starknet-devnet-types/src/rpc/transactions/l1_handler_transaction.rs

```diff
@@ -154,6 +180,15 @@ mod tests {
         )
         .unwrap();
 
+        // message hash string taken from:
+        //  https://testnet.starkscan.co/tx/0x06182c63599a9638272f1ce5b5cadabece9c81c2d2b8f88ab7a294472b8fce8b#messagelogs
+
+        assert_eq!(
```

Comment:
Why is this assertion now necessary?

Reply:
to check if message hash is calculated correctly

Reply:
Later I realized that there is this part of the PR description:
> Added message_hash to L1HandlerTransactionReceipt